code/datums/abilities/vampire.dm

```diff
@@ -409,7 +409,6 @@
 				VZ.master = src
 
 			boutput(M, "<span class='alert'><b>You awaken filled with purpose - you must serve your master vampire, [owner.real_name]!</B></span>")
-			M.show_antag_popup("mindhack")
```

Comment:
no thrall popup?

Reply:
The thrall popup is handled further down in the code, this line just made the mindhack one pop up first and override the thrall one.